src/components/views/ViewDataTable/ViewDataTableToolbar.tsx

```diff
@@ -1,16 +1,34 @@
-import { Add } from '@material-ui/icons';
 import { FormattedMessage } from 'react-intl';
+import { Add, Launch } from '@material-ui/icons';
 import { Box, Button } from '@material-ui/core';
 
+
 export interface ViewDataTableToolbarProps {
     onColumnCreate: () => void;
+    onViewCreate: () => void;
+    selection: number[];
 }
 
-const ViewDataTableToolbar: React.FunctionComponent<ViewDataTableToolbarProps> = ({ onColumnCreate }) => {
+const ViewDataTableToolbar: React.FunctionComponent<ViewDataTableToolbarProps> = ({
+    onColumnCreate,
+    onViewCreate,
+    selection,
+}) => {
     return (
         <Box display="flex" justifyContent="flex-end">
-            <Button data-testid="create-column-button" onClick={ onColumnCreate }>
-                <Add /><FormattedMessage id="misc.views.createColumn" />
+            { !!selection.length && (
```

Comment:
maybe this would be nicer if it were always rendered, but disabled if there is no selection? maybe a tooltip informing users they need to select some rows to use this?

Reply:
I think that's a nice idea. I did it this way because that's the way it was designed in the wireframes.

The problem with either of those solutions though is that we will be running out of space in the toolbar before too long, so I don't think either solution will actually be very sustainable. For that reason, I'd rather not spend more time on this right now before we've implemented all the toolbar features and can get a feel for what to do in a more general sense. I suspect we will need to rework it a little bit.